Core/Meta/Extensions/StringExtensions.cs

```diff
@@ -484,7 +484,7 @@ public static string EscapeString(this string? value)
         [GeneratedRegex(@"^.*[\*\?\[\]].*(\.[a-zA-Z0-9]+)?$")]
         private static partial Regex LegalFileGlobRegex();
 
-        [GeneratedRegex(@"^[a-zA-Z]+(\.[a-zA-Z]+)*$")]
+        [GeneratedRegex(@"^[a-zA-Z]+(\.[a-zA-Z]+|::[a-zA-Z]+)*$")]
```

Comment:
you'll need to verify if namespace can still be passed on the commanline with this change 

bebopc build -g ""cpp:./GeneratedTestCode/test.g.hpp,namespace=Bebop::Codegen"

Reply:
I've used this build in the combination of cmake, it works flawlessly

```cmake
bebop_generate(Schema-Models
    VERSION "v3.0.14"
    LANGUAGE cpp
    OUTPUT ${SCHEMA_MESSAGE_OUTPUT}
    BOPS ${SCHEMA_MODELS}
    OPTIONS "namespace=Game::Serializable::Models"
)
```